server/api/routes/users.js

```diff
@@ -1,8 +1,11 @@
 const express = require('express');
+const subscriptions = require('./subscriptions');
 const { getUsers, createUser } = require('../actions/users');
 
 const router = express.Router();
 
+router.use('/', subscriptions);
```

Comment:
this should have `subscriptions` here as the URL prepend for all routes in the subscriptions file

```suggestion
router.use('/subscriptions', subscriptions);
```